### PR TITLE
cleate a Lottery class with getRandomBall method

### PR DESCRIPTION
**Pull request by inbox-reva**
Hi! Please check my solution.

### CODE REVIEW

File: src/main/java/core/basesyntax/Application.java [hunk 1]
```diff
@@ -3,5 +3,9 @@
 public class Application {
     public static void main(String[] args) {
         // create three balls using class Lottery and print information about them in console
+        Lottery lottery = new Lottery();
+        for (int i = 0; i < 3; i++) {
+            System.out.println(lottery.getRandomBall().toString());
```
Comment (Rommelua) [inline src/main/java/core/basesyntax/Application.java:8]:
```suggestion
            System.out.println(lottery.getRandomBall());
```
toString() will be called automatically

File: src/main/java/core/basesyntax/ColorSupplier.java [hunk 2]
```diff
@@ -1,7 +1,10 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
-    public String getRandomColor() {
-        return null;
+    public Color getRandomColor() {
+        int index = new Random().nextInt(Color.values().length);
```
Comment (Rommelua) [inline src/main/java/core/basesyntax/ColorSupplier.java:7]: Don't create several instances of the same class if you can use only one instance for your purposes
You create new Random() each time when getRandomColor() is called. Make Random class field

File: src/main/java/core/basesyntax/Lottery.java [hunk 3]
```diff
@@ -0,0 +1,12 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+
```
Comment (Rommelua) [inline src/main/java/core/basesyntax/Lottery.java:6]: Please don’t add redundant empty lines to your code.

File: src/main/java/core/basesyntax/Lottery.java [hunk 4]
```diff
@@ -0,0 +1,12 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+
+    public Ball getRandomBall() {
+        Color color = new ColorSupplier().getRandomColor();
+        int number = new Random().nextInt(100);
```
Comment (Rommelua) [inline src/main/java/core/basesyntax/Lottery.java:9]: 1. Don't create several instances of the same class if you can use only one instance for your purposes
2. 
image

**Review by Rommelua — CHANGES_REQUESTED** [review]

File: src/main/java/core/basesyntax/Lottery.java [hunk 5]
```diff
@@ -0,0 +1,14 @@
...
+
+import java.util.Random;
+
+public class Lottery {
+    private ColorSupplier colorSupplier = new ColorSupplier();
+    private Random rundom = new Random();
+
+    public Ball getRandomBall() {
+        Color color = colorSupplier.getRandomColor();
+        int number = rundom.nextInt(100);
```
Comment (aaananas) [inline src/main/java/core/basesyntax/Lottery.java:11]: avoid magic numbers - use constants

**Review by aaananas — APPROVED** [review]

**Pull request closed without merging**